.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+repos:
```

Comment:
Can we remove the precommit-hook here ?

Reply:
Yes, but I cannot it's automated tools so you can do it with your collaborator right